indico/modules/events/registration/clone.py

```diff
@@ -120,6 +120,8 @@ def _has_content(self, event):
                 .has_rows())
 
     def _clone_registrations(self, old_form, new_form, field_data_map):
+        self._delete_orphaned_registrations(new_form.event_id)
```

Comment:
We don't need to call this when doing a normal clone, just when doing an import (`event_exists` being `True` in `run`).